streampark-common/src/main/java/org/apache/streampark/common/enums/ClusterState.java

```diff
@@ -24,12 +24,21 @@ public enum ClusterState implements Serializable {
   /** The cluster was just created but not started */
   CREATED(0),
   /** cluster started */
-  STARTED(1),
+  RUNNING(1),
   /** cluster stopped */
   STOPPED(2),
-
   /** cluster lost */
-  LOST(3);
+  LOST(3),
+  /** cluster undefined */
+  UNDEFINED(4),
+  /** cluster successfully */
+  SUCCEEDED(5),
```

Comment:
The two enum look like redundant on platform side.
Its could be used to represent the final status on yarn just platform side.

Reply:
When coding, I considered redefining an enumeration class to represent the obtained yarn final status, but later found that adding an enumeration class had redundancy in the status of the platform side. Therefore, I defined the status information of yarn final status in the platform side. Because I think this is non conflicting, and the platform side also needs these states to represent the running status of the flink cluster.

Reply:
> The two enum look like redundant on platform side. Its could be used to represent the final status on yarn just platform side.

When coding, I considered redefining an enumeration class to represent the obtained yarn final status, but later found that adding an enumeration class had redundancy in the status of the platform side. Therefore, I defined the status information of yarn final status in the platform side. Because I think this is non conflicting, and the platform side also needs these states to represent the running status of the flink cluster.

Reply:
Sorry for the  not clear statements.
I want to express  that platform just focus the status about flink clusters abstraction of platform and should ignore the real status of finalStatus on yarn side. or in other words, the finalStatus is hidden for the Enum.